example/ios/Podfile

```diff
@@ -32,15 +22,12 @@ target 'KlaviyoReactNativeSdkExample' do
 
   use_react_native!(
     :path => config[:reactNativePath],
-    # Enables Flipper.
-    #
-    # Note that if you have use_frameworks! enabled, Flipper will not work and
-    # you should disable the next line.
-#    :flipper_configuration => flipper_config,
-    # An absolute path to your application root.
     :app_path => "#{Pod::Config.instance.installation_root}/.."
   )
 
+  #TODO: remove this once iOS modularization release is made and the podspec is updated
+  pod 'KlaviyoSwift', :path => '../../../klaviyo-swift-sdk/'
```

Comment:
I think build is failing because of this.

Reply:
@ajaysubra @ndurell are we concerned about the failing build at the moment? This change is only temporary so we can work locally until the Swift SDK v4.0.0 is released. Then we revert the changes and CI should build again.

If we want to get CI to pass we can point to the remote git branch rather than the relative path. But then when we're working on it locally we'll probably want to change it back to the relative path

Reply:
Uhh do we have to merge with this in? I'd prefer to keep master in a non broken state if possible.

Reply:
Yeah you are right. Removed it 🤞🏾

Reply:
thanks for checking @ndurell  and @ab1470

Reply:
So the build is still failing. Went on a wild goose chase and found out that even with this line removed the build will fail because the Event name changes in here are not in the release swift version that the podspec is pointing to. Unfortunately, the podspec can't point to a branch of the swift SDK. 
```
2024-09-24T02:50:43.2997240Z ** BUILD FAILED **
2024-09-24T02:50:43.2997360Z 
2024-09-24T02:50:43.2997370Z 
2024-09-24T02:50:43.2997490Z The following build commands failed:
2024-09-24T02:50:43.2998390Z 	SwiftCompile normal x86_64 /Users/runner/work/klaviyo-react-native-sdk/klaviyo-react-native-sdk/ios/KlaviyoBridge.swift (in target 'klaviyo-react-native-sdk' from project 'Pods')
2024-09-24T02:50:43.2999890Z 	SwiftCompile normal x86_64 Compiling\ KlaviyoBridge.swift /Users/runner/work/klaviyo-react-native-sdk/klaviyo-react-native-sdk/ios/KlaviyoBridge.swift (in target 'klaviyo-react-native-sdk' from project 'Pods')
2024-09-24T02:50:43.3000780Z (2 failures)
2024-09-24T02:50:43.3001900Z [ERROR] command finished with error: command (/Users/runner/work/klaviyo-react-native-sdk/klaviyo-react-native-sdk/example) /private/var/folders/4h/j1rdr7ws1cbck70c6bjbk5t40000gn/T/xfs-f23266fe/yarn run build:ios exited (65)
```

I suggest we keep this line in and merge mainly because we could at least work on this locally and when we release iOS we can get this build passing again. Long term with a WIP swift/android branch we need to figure out how to work.